<!DOCTYPE HTML>
<html lang="en">
  <head>
  	<title>Daniel Marquina - Research</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700,800,900" rel="stylesheet">
		
		<link rel="stylesheet" href="css/font-awesome.min.css">
		<link rel="stylesheet" href="css/style.css">
		<link rel="stylesheet" href="css/academicons.min.css"/>
  </head>
  <body>
		
		<div class="wrapper d-flex align-items-stretch">
			<nav id="sidebar">
	  		<div class="img bg-wrap text-center py-4" style="background-image: url(images/barcode.jpeg);">
	  			<div class="user-logo">
	  				<div class="img" style="background-image: url(images/logo.jpg);"></div>
	  				<h3 style="font-size:26px;background-color: white;color:black">Daniel Marquina, PhD</h3>
	  			</div>
	  		</div>
        <ul class="list-unstyled components mb-5">
          <li align="center">
            <a href="index.html">About</a>
          </li>
          <li align="center">
              <a href="research.html">Research</a>
          </li>
          <li align="center">
            <a href="curriculum.html">CV</a>
          </li>
          <li align="center">
            <a href="publications.html">Publications</a>
          </li>
          <br>
            &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://scholar.google.es/citations?user=sCNxzWAAAAAJ&hl=es&oi=ao" class="ai ai-google-scholar ai-2x" style="color:white" target="_blank"></a>
            &nbsp; &nbsp;<a href="https://www.researchgate.net/profile/Daniel_Marquina" class="ai ai-researchgate ai-2x" style="color:white" target="_blank"></a>
            &nbsp; &nbsp;<a href="http://orcid.org/0000-0001-5722-058X" class="ai ai-orcid ai-2x" style="color:white" target="_blank"></a>
            &nbsp; &nbsp;<a href="https://twitter.com/metagusano" class="fa fa-twitter fa-2x" style="color:white" target="_blank"></a>
            &nbsp; &nbsp;<a href="https://www.linkedin.com/in/daniel-marquina-0234b0130/" class="fa fa-linkedin fa-2x" style="color:white" target="_blank"></a>
            &nbsp; &nbsp;<a href="https://github.com/metagusano" class="fa fa-github fa-2x" style="color:white" target="_blank"></a>
            &nbsp;&nbsp;&nbsp;<a href="mailto:danielmarquinahz@gmail.com" class="fa fa-envelope fa-2x" style="color:white" target="_blank"></a>
        </ul>
    	</nav>

		<!-- Tittle -->
    	<section id="content"  class="p-5 p-md-5 pt-5">
					<div class="p-5 p-md-5 pt-5">
						<h2>Research</h2>
					</div>
					<div class="p-5">
					<p style="font-family:arial;font-size:16px" align="justify">I have marveled at Earth's biodiversity for as long as I can remember. For my professional career, I am interested in understanding how that biodiversity is generated and articulated in ecosystems. This means identifying and documenting the species that compose them, how they originated, and how they interact with each other. To accomplish such a task it is necessary to work with a variety of disciplines, from taxonomy, to evolutionary biology, or quantitative ecology.
					<br><br>As a molecular ecologist, I study ecosystems from the viewpoint of their smallest component: DNA. To do so I apply molecular techniques that rely on DNA sequencing, mainly (but not only) DNA metabarcoding. Metabarcoding is a technique that allows the simultaneous identification of species from a mixed sample by the amplification and examination of a short genetic sequence. These mixed samples can be bulk samples (for instance, an insect trap or a plankton tow net sample), environmental DNA samples (eDNA, like water from a river or soil), and other diverse types of samples (scats, gut contents, air filtering samples, etc.). Compared to traditional methods of identification, metabarcoding is faster, has higher taxonomic resolution, and can be used with samples in which the individuals are not physically present, or are degraded.</p>
					<div class="pt-5">
						<h2 style="font-family:arial;font-size:26px">Current Projects:</h2>
					</div>
					<p style="font-family:arial;font-size:15px" align="justify">Currently I am not in a reserach-related position, although I work with reserach and reserachers every day. However, I am always open for interesting collaborations relating biodiversity, ecology, and metabarcoding.</p>
					<div class="pt-5">
						<h2 style="font-family:arial;font-size:26px">Past Projects:</h2>
					</div>
					<br>
					<div class="pt-5">
					<h4 style="font-family:arial"><strong>Simultaneous assessment of the symbiome of insect species of natural and economic interest</strong></h4>
					<p style="font-family:arial;font-size:15px" align="justify"><img class="rounded float-right" src="images/EAB.png" alt="EAB" width= "350" height= "350"/>Ecosystems are made of the species present in them and the interactions these species establish with each other. When an organism interacts with another (for instance, when a spider eats a fly, or a bee visits a flower), it leaves DNA traces that can be analysed. Not only that, but each individual is hosting a plethora of carry-on organisms: bacteria, parasites, fungi, etc. It is possible, with metabarcoding, to obtain the information of what other organisms an individual has interacted with, but it is technically challenging due to the huge diversity of these organisms, and costly.
					<br><br>The main objective of this project was to advance in the laboratory and bioinformatic methodology to facilitate the charting of interaction networks of a series of insect species of natural and commercial interest: forest pests and solitary bees. Selection of the best genetic markers, multiplex PCR (amplification of several markers from different organisms in the same reaction tube), and adequate bioinformatic pipelines were among the challenging tasks of this project.
					<br><br>More work is indeed needed to refine the methods so we can robustly and confidently use metabarcoding for reconstructing wide-range ecological interactions of a species.</p>
					<br>
					<br>
					<h4 style="font-family:arial"><strong>Metabarcoding of insects for accelerated taxonomic discovery</strong></h4>
					<p style="font-family:arial;font-size:15px" align="justify"><img class="rounded float-left" src="images/MTsample.jpg" alt="MTsample" width= "350" height= "350"/>Metabarcoding can be greatly beneficial to insect biodiversity research by speeding up the process of documenting the entomofauna of any given habitat, quickly generating species inventories of large areas, or finding new and unknown species in mass-sampling catches. However, the laboratory and bioinformatic protocols for metabarcoding of insects still require some optimisation.
					<br><br>During my PhD I worked on optimising some aspects of the metabarcoding workflow to be applied to tackle this problem. First, I evaluated different genetic markers to identified the best-performing one, and I designed new primers for the mitochondrial rRNA gene 16S that outperforms previously designed ones. I also investigated how using different markers and sample types simultaneously increase biodiversity detection. More specifically, I tested the potential of extracting insects' DNA from the preservative ethanol used in the sampling. Since that didn't result in a viable alternative to the destruction of the insects for the extraction of the DNA, I evaluated different techniques for obtaining the DNA from bulk samples of insects in a non-destructive way, so the catched individuals could be later examined by taxonomists to identify and describe new species.
					<br><br>Although this was done using terrestrial insects, many of the protocols and results of this project can be applied to other taxonomic gruops and sample types. This project gave me the opportunity as well to collaborate with other research teams, taxonomists and ecologists alike.</p>
					<br>
					<br>
					<h4 style="font-family:arial"><strong>Taxonomy, systematics, and evolution of polyclad flatworms</strong></h4>
					<p style="font-family:arial;font-size:15px" align="justify"><img class="rounded float-right" src="images/yungia.jpg" alt="yungia" width= "430" height= "290"/>Polycladida is one of the two macroscopic orders of free-living Platyhelminthes. They inhabit all oceans of the planet, from the intertidal to the deep sea (more than 2500 metres deep), but they are most diverse and conspicuous in tropical waters. They are known to be predators, some more specialized, but other some are voracious generalists, although their role in ecosystems is poorly studied. There are around 1100 described species, but still a lot of its diversity is unknown. This is a neglected taxonomic group with very few expert taxonomists worldwide.
					<br><br>A significant part of my research was focused on describing the biodiversity of Polycladida, mainly in the Iberian Peninsula, but also in other areas such as the Caribbean Sea or the Great Barrier Reef (Australia). This was mostly a taxonomic work that substantially increased the knowledge of the order for Spanish waters. The study included collection, histological preparation, morphological examination (primarly of the reproductive structures) and description. Together with my co-authors I described and named new species (<i>Notoplanella estelae</i> and <i>Lurymare clavocapitata</i> amongst them) from the Mediterranean Sea, and the Atlantic and Pacific Oceans, belonging to both suborders Cotylea and Acotylea. In addition, I analysed other aspects of the morphology, from a comparative perspective, to elucidate how evolution has shaped this organisms.
					<br><br>We also focused on the phylogeny of the whole group and of some specific taxa within, as well as on other evolutionary processes, using molecular biology. We designed order- and species-specific PCR primers targeting different regions of the mitochondrial genomes, and reconstructed the phylogeny of Polycladida based on them, being one of the few studies aimed at the evolutionary history of the whole group. Finally, I completed my master's thesis on species delimitation in the genera <i>Pseudoceros</i> and <i>Pseudobiceros</i>, using morphological examination and DNA barcoding.</p>
					<p style="font-size:10px; color:grey"> &copy; <i>Yungia aurantiaca</i> picture by Francisco Sedano <a href="https://www.flickr.com/photos/90784457@N04/">(flickr here)</a>.</p>
          				<br>
					<br>
					<h4 style="font-family:arial"><strong>Spatial ecology of meiofauna and their potential as bioindicators</strong></h4>
					<p style="font-family:arial;font-size:15px" align="justify"><img class="rounded float-left" src="images/kino.jpg" alt="yungia" width= "430" height= "290"/>Meiofauna is composed by small animals, ranging from 42 to 500 Î¼m in lenght, that live in between the grains of sediment (sand, mud, etc.) in aquatic environments (from beaches to freshwater to deep sea). Twenty out of the thirty-four animal phyla have representatives in meiofauna, and five of them (Kinorhyncha, Gastrotricha, Loricifera, Priapulida and Gnathostomulida) are exclusive from it. Given the difficulty of studying them, due to the small body size or complicated sampling, some of the meiofaunal groups are vastly understudied. Most of the organisms present in the meiofauna community spend most part, if not the entirety, of their life cycle in the sediment. Some feed on bacteria or microalgae (e.g. kinorhynchs), while other are predators (e.g. tardigrades or nematodes) of their coinhabitants. Either way, they represent an excelent system to examine the changes that undergo in an environment due to human pressure.
					<br><br>This was a short project in collaboration with Francisco Sedano and Free Espinosa from the University of Sevilla, conducted during my last year of bachelor. We studied how the differences in sediment granulometry and depth affect the composition of the meiofaunal community in the inter/subtidal area, in order to later compare protected and anthropized areas. We studied how the proportion of the different taxonomic groups changes from coarse to fine sediment, and from 5 m to 15 m deep. Also, we compared the meiofaunal community of an underwater nature reserve and a nearby marina, along with analysis of usual organic and inorganic pollutants.</p>
			</div>
			
		<!-- Footer -->

      <footer id="footer">
				<div class="inner">
					<div class="flex-2">
				    <article>
				      <div class="copyright">
							  &copy; 2021. All rights reserved
					    </div>
				    </article>
			    </div>
			   </div>
		</footer>
		</section>
		</div>
  </body>
</html>
